support-frontend/assets/helpers/supporterPlus/benefitsThreshold.ts

```diff
@@ -20,7 +24,23 @@ export function getLowerProductBenefitThreshold(
 			? 'DomesticAnnual'
 			: 'DomesticMonthly';
 	const ratePlanRegularContribution = getBillingPeriodTitle(billingPeriod);
-	const ratePlan =
-		product === 'TierThree' ? ratePlanTier3 : ratePlanRegularContribution;
-	return productCatalog[product]?.ratePlans[ratePlan]?.pricing[currencyId] ?? 0;
+
+	if (product === 'TierThree') {
+		return (
+			productCatalog[product]?.ratePlans[ratePlanTier3]?.pricing[currencyId] ??
+			0
+		);
+	}
+
+	if (product === 'HomeDelivery' || product === 'SubscriptionCard') {
+		return (
+			productCatalog[product]?.ratePlans[ratePlan]?.pricing[currencyId] ?? 0
+		);
+	}
+
+	return (
+		productCatalog[product]?.ratePlans[ratePlanRegularContribution]?.pricing[
```

Comment:
Same here, pull const `ratePlanRegularContribution` through here as only needed in this block?